### DISCUSSION TITLE
require("nvchad_ui.tabufline") is not working

### DISCUSSION THREAD

**Discussion by benzwt** [NvChad/NvChad#2284] [Q&A] [answered]
was there any big changes in nvchad_ui ?
I just pull  d3d9aa251a9dd94881cdbc4 from repos.
I found that 
>    require("nvchad_ui.tabufline") 
is not working

here is the error messages:
```
E5108: Error executing lua: /home/xxx/.config/nvim/lua/custom/mappings.lua:83: module 'nvchad_ui.tabufline' not found:
        no field package.preload['nvchad_ui.tabufline']
cache_loader: module nvchad_ui.tabufline not found
cache_loader_lib: module nvchad_ui.tabufline not found
```
Please shade some light for me to solve this out.
thanks!

**Comment by siduck** [comment, marked as answer]
remove _ui , update nvchad too
